### PR TITLE
CODE RUB: Run code cleanup

### PR DESCRIPTION
**Pull request by geetika-jain**

### CODE REVIEW

File: DMX.Core.Api.Tests.Acceptance/APIs/LabCommands/LabCommandApiTests.cs [hunk 1]
```diff
@@ -38,7 +36,7 @@ private static LabCommand UpdateRandomLabCommand(LabCommand inputLabCommand)
                 .OnProperty(labCommand => labCommand.Id).Use(inputLabCommand.Id)
                 .OnProperty(labCommand => labCommand.CreatedDate).Use(inputLabCommand.CreatedDate)
                 .OnProperty(labCommand => labCommand.UpdatedDate).Use(now);
-                //.OnType<DateTimeOffset>().Use(GetRandomDateTime());
+            //.OnType<DateTimeOffset>().Use(GetRandomDateTime());
```
Comment (hassanhabib) [inline DMX.Core.Api.Tests.Acceptance/APIs/LabCommands/LabCommandApiTests.cs:39]: No comments allowed

Comment (geetika-jain) [reply DMX.Core.Api.Tests.Acceptance/APIs/LabCommands/LabCommandApiTests.cs:39]: I remember fixing this in one of the PRs

Comment (hassanhabib) [reply DMX.Core.Api.Tests.Acceptance/APIs/LabCommands/LabCommandApiTests.cs:39]: @geetika-jain there's still a comment here

Comment (geetika-jain) [reply DMX.Core.Api.Tests.Acceptance/APIs/LabCommands/LabCommandApiTests.cs]: Yeah I forgot to push :/

**Review by hassanhabib — APPROVED** [review]

**Comment by hassanhabib** [comment]
@geetika-jain merge with master

**Pull request merged**